test/parallel/test-os.js

```diff
@@ -1,74 +1,88 @@
 'use strict';
-var common = require('../common');
-var assert = require('assert');
-var os = require('os');
-var path = require('path');
-
+const common = require('../common');
+const assert = require('assert');
+const os = require('os');
+const path = require('path');
+
+const is = {
+  string: (value) => { assert.ok(typeof value === 'string'); },
+  number: (value) => { assert.ok(typeof value === 'number'); },
+  array: (value) => { assert.ok(Array.isArray(value)); },
+  object: (value) => { assert.ok(typeof value === 'object'); }
```

Comment:
Sure, originally I wrote the not null test here, but since `typeof null === 'object'` I thought the `is.object` with this check was a nonsense.
Anyhow, now I added it again, let me know if something else is missing :)